portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,7 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function navtoggle() {
+  document.getElementById("navmenu").classList.toggle("visible");
```

Comment:
Please use single quotes rather than double quotes for javascript string literals per our style guide:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes